project-plan.md

```diff
@@ -0,0 +1,189 @@
+# 1. Mål og rammer
+
+## 1.1 Bakgrunn
+
+*See [project-description.md](./project-description.md)*
+
+## 1.2 Prosjektmål
+
+Develop a proof-of-concept SSI application, and use the proof-of-concept to discover and present strengths and weaknesses of the current SSI infrastructure, with a focus on interoperability. 
+
+There are two key areas of interoperability:
+1. Interoperability between different ledgers - where DIDs and VCs are stored.
+2. Interoperability between different wallets - where the ownership of DIDs and VCs are prooved.
+
+## 1.3 Rammer
+
+The BSc project runs between `01.feb.2021` and `20.may.2021`.
+
+The BSc project will be developed as an open source project on the DIN-Foundation's Github page here: https://github.com/DIN-Foundation/bsc-ntnu-2020/. 
+
+See https://www.din.foundation/ for more info.
+
+
+# 2. Omfang
+
+## 2.1 Fagområde
+
+The project is developed within the field of Self-sovereign-identity, aka SSI.
+
+## 2.2 Avgrensing
+
+The project is looking at interoperability of the current SSI-infrastructure. More specifically interoperability between wallets, and interoperability between ledgers.
+
+To understand interoperability a proof-of-concept application will be developed. The proof-of-concept will implement one specific use-case of SSI. 
+
+The proof-of-concept should implement the following parts of an SSI-workflow:
+- Issuing a credential to a ledger
```

Comment:
You dont issue a credential to a ledger :P You issue to a holder, who has a DID, on whatever ledger

Reply:
Addressed in 5099505